src/main/resources/assets/js/app/wizard/ContentWizardPanel.ts

```diff
@@ -1823,41 +1824,48 @@ export class ContentWizardPanel
         return Q.all(formViewLayoutPromises).thenResolve(null);
     }
 
-    private toggleElementsVisibility(visible: boolean) {
+    private toggleSettingsElementsVisibility(visible: boolean) {
         this.settingsWizardStepForm.setVisible(visible);
         this.settingsWizardStepForm.getPreviousElement().setVisible(visible);
         this.settingsWizardStep.getTabBarItem().setVisible(visible);
-        this.editPermissionsToolbarButton.setVisible(visible);
     }
 
-    private toggleStepFormsVisibility(loginResult: LoginResult) {
-        const hasAdminPermissions: boolean = this.hasAdminPermissions(loginResult);
+    private toggleStepFormsVisibility() {
+        const hasAdminPermissions: boolean = this.hasAdminPermissions();
 
         if (hasAdminPermissions) {
-            this.toggleElementsVisibility(true);
+            this.toggleSettingsElementsVisibility(true);
+            this.editPermissionsToolbarButton.setVisible(true);
         } else {
-            ProjectHelper.isUserProjectOwner(loginResult).then((isOwner: boolean) => this.toggleElementsVisibility(isOwner));
+            ProjectHelper.isUserProjectOwner(this.loginResult).then((isOwner: boolean) => {
+                const isContentExpert: boolean = this.loginResult.isContentExpert();
+
+                this.toggleSettingsElementsVisibility(isContentExpert && isOwner);
```

Comment:
This is incorrect. A project owner doesn't need Content Expert role in order to see Settings.